src/Command/DeleteFlagsHandler.php

```diff
@@ -48,8 +49,13 @@ public function handle(DeleteFlags $command)
 
         $actor->assertCan('viewFlags', $post->discussion);
 
+        // remove beta 17
         $this->events->dispatch(new FlagsWillBeDeleted($post, $actor, $command->data));
 
+        foreach ($post->flags as $flag) {
+            $this->events->dispatch(new Deleting($flag, $actor));
```

Comment:
My example use case for individual deleted flags is this (which I also hope to build as an extension if this PR is accepted):

Each `Created` event is captured and stored someplace. To complete the flag lifecycle from creation to closure, an individual `Delet(ing/ed)` matching event would be required, rather than a general 'all flags for this post were cleared'. 

Take the general concept of my previous flag dismiss logging. Obviously that would track Mods (or whoever was granted permissions) to dismiss flags and log/keep records of those actions for audit trail purposes. Go one step further than that, say you had a need to build some statistical analysis patterns on how many flags a user has created, or to build up some ML over which type of posts get flagged by which user type, or simply to build an average on how long flags remain open before being resolved.

Long story short, the beginning and end events of an action should match, I believe.